src/Lykke.Service.PayInternal.Core/Domain/PaymentRequests/PaymentRequest.cs

```diff
@@ -59,5 +59,7 @@ public bool StatusValidForSettlement()
         }
 
         public string Initiator { get; set; }
+
+        public string SettlementErrorDescription { get; set; }
```

Comment:
I believe ProcessingError property is intended to be used for this purpose. Please consider the possibility of using it.